src/google/protobuf/message.h

```diff
@@ -632,7 +632,7 @@ class PROTOBUF_EXPORT Reflection final {
     friend class Reflection;
 
     absl::string_view CopyFromCord(const absl::Cord& cord) {
-      if (absl::optional<absl::string_view> flat = cord.TryFlat()) {
+      if (std::optional<absl::string_view> flat = cord.TryFlat()) {
```

Comment:
hmm our cmake tests are failing here because absl::Cord::TryFlat is returning an absl::optional.  We're likely using an older version of Abseil in our CI, so I'd have to double check what our compatibility window is.  Either we can upgrade that and push this through, or we'll have to wait until we drop support for that version of Abseil

Reply:
clever, if this is the only instance I think I can live with `auto` :)